eZ/Publish/API/Repository/Tests/ContentServiceTest.php

```diff
@@ -6424,4 +6425,50 @@ private function createContentWithReverseRelations(array $drafts)
 
         return $contentWithReverseRelations;
     }
+
+    public function testPublishVersionByUserWithLanguageLimitation()
```

Comment:
Please:
1. Move this test to a dedicated test case, e.g.: `\eZ\Publish\API\Repository\Tests\Limitation\LanguageLimitationTest::testPublishVersionTranslation`.

2. Add `: void` return type.

3. Create a negative test case - publishing explicit translations list, but for a user who shouldn't be allowed to do so (e.g. trying to publish English, while having limitation to German).